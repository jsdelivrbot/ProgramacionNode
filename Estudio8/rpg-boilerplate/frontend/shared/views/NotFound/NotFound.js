'use strict'
import React from 'react'
import createReactClass from 'create-react-class'

const NotFound = createReactClass({
  render () {
    return (<div className='page-title'>Page Not Found</div>)
  }
})

export default NotFound
