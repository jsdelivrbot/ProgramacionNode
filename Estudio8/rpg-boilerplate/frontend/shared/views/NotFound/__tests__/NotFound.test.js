/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import NotFound from '../NotFound'

describe('A </NotFound>', function () {
  it('renders correctly', () => {
    expect(toJson(shallow(<NotFound />))).toMatchSnapshot()
  })
})
