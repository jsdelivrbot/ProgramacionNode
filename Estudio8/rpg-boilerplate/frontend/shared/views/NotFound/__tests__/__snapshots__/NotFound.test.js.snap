exports[`A </NotFound> renders correctly 1`] = `
<withTheme(AppPrivate)
  noHeader={true}>
  <withStyles(Card)
    className="not-found">
    <div
      className="page-title">
      Page Not Found
    </div>
  </withStyles(Card)>
</withTheme(AppPrivate)>
`;
